tests/run/round.py

```diff
@@ -0,0 +1,83 @@
+# mode: run
+
+import cython
+
+
+@cython.cfunc
+@cython.returns(cython.float)
+def round_float(value: cython.float):
+    return round(value)
+
+
+@cython.cfunc
+@cython.returns(cython.double)
+def round_double(value: cython.double):
+    return round(value)
+
+
+@cython.cfunc
+@cython.returns(cython.longdouble)
+def round_long_double(value: cython.longdouble):
+    return round(value)
```

Comment:
I'd merge these back into the test functions below.
I'd also add tree assertions that the optimisation was actually applied, i.e. that there is no longer a `SimpleCallNode` but a `PythonCapiCallNode` in the syntax tree.